OSDOCS Speeding Up Pulling Container Images/CRI-O Additional Storage Support (#110809)
openshift/openshift-docs: nodes/nodes/nodes-nodes-additional-crio-storage.adoc

:_mod-docs-content-type: ASSEMBLY
[id="nodes-nodes-additional-crio-storage"]
= Additional CRI-O storage locations for faster container startup
include::_attributes/common-attributes.adoc[]

> :context: adding-node-iso

Reviewer: I think that's the wrong context, should it be?

```suggestion
:context: nodes-nodes-additional-crio-storage
```

:_mod-docs-content-type: ASSEMBLY
[id="nodes-nodes-additional-crio-storage"]
= Additional CRI-O storage locations for faster container startup
include::_attributes/common-attributes.adoc[]
:context: nodes-nodes-additional-crio-storage

toc::[]

[role="_abstract"]

> You can configure additional storage locations for the CRI-O container engine by using the `ContainerRuntimeConfig` custom resource (CR). Three new fields let you specify where CRI-O stores and resolves container image layers, complete container images, and OCI artifacts. By specifying locations other than the default, you can reduce application startup time, make your applications run more efficiently, and configure lazy pulling. 

Reviewer 2: shortdesc must be 50-300 characters, this is 435. Maybe:

```suggestion
To reduce application startup time, make your applications run more efficiently, and configure lazy pulling, you can configure additional storage locations for the CRI-O container engine. 

Fields in the `ContainerRuntimeConfig` custom resource (CR) let you specify where CRI-O stores and resolves container image layers, complete container images, and OCI artifacts.
```

(that combined is still 366, so would need a second paragraph to include the second part as context)
